src/main/resources/META-INF/plugin.xml

```diff
@@ -201,6 +201,11 @@
                     icon="com.virtuslab.gitmachete.frontend.icons.MacheteIcons.HELP">
                 <override-text place="GitMacheteToolbar"/>
             </action>
+
+            <action id="GitMachete.TraverseRepositoryAction"
```

Comment:
probably it's too soon to assign anything specific but a key shortcut for this one might be very useful